python_modules/dagster/dagster/_core/definitions/run_status_sensor_definition.py

```diff
@@ -253,10 +253,33 @@ class RunFailureSensorContext(RunStatusSensorContext):
         failure_event (DagsterEvent): the pipeline failure event.
```

Comment:
```suggestion
        failure_event (DagsterEvent): the run failure event.
```
nit - make consistent with the property doc string

Reply:
can't make a comment on the above line, but could also change 
```
dagster_run (DagsterRun): the failed pipeline run.
```
to 
```
dagster_run (DagsterRun): the failed run.
```